src/Cart.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState, useRef } from "react";
+import pizzaImg from './assets/pizza.jpg'
+
+
+function Cart(props) {
+
```

Comment:
Perhaps some of the blank lines could be deleted to make the code a little leaner?